src/views/Home/components/PrizeTracks/PrizeTracks.scss

```diff
@@ -0,0 +1,107 @@
+@use "~stylesheets/typography";
+
+.prizeTracks {
+  background: linear-gradient(#b0b6f7, #b0b6f7);
```

Comment:
I think the background is currently set to what the component above and below in this build. That's going to change once we get the other components merged in so I would update this based on the figma colors.